src/attribute.ts

```diff
@@ -0,0 +1,32 @@
+import * as path from "path";
+import { Position, Range, Uri, window } from "vscode";
+import {
+  Capability,
+  isAnyCapabilityEnabled,
+} from "./capabilities/capabilities";
+import { setDecoration } from "./vscode.api";
+
+const logoPath = Uri.file(path.resolve(__dirname, "..", "small_logo-mono.png"));
+
+const type = window.createTextEditorDecorationType({
+  gutterIconPath: logoPath,
+  gutterIconSize: "70%",
+});
+
+export default function attribute(position: Position): void {
+  if (
+    isAnyCapabilityEnabled(Capability.ATTRIBUTION, Capability.ALPHA_CAPABILITY)
+  ) {
+    drawDecorationAtCurrentLine(position);
+  }
+}
+
+window.onDidChangeTextEditorSelection(clearDecoration);
```

Comment:
This is a bit side effect-ish. no?

Reply:
not a bit, it is a side effect. but what is your question /suggestion here?

Reply:
Add it under method and call it somewhere..